.github/workflows/perl-openwsman.yml

```diff
@@ -138,8 +179,7 @@ jobs:
           rpm_gpg_signing_passphrase: ${{ secrets.RPM_GPG_SIGNING_PASSPHRASE }}
 
       # set condition to true if artifacts are needed
-      - if: ${{ false }}
-        name: Upload package artifacts
+      - name: Upload package artifacts
```

Comment:
@kduret, shouldn't we keep this condition to avoid to always upload the artifact ?

Reply:
Yes indeed